infra/main.bicep

```diff
@@ -283,7 +293,7 @@ module backend 'core/host/appservice.bicep' = {
     managedIdentity: true
     virtualNetworkSubnetId: isolation.outputs.appSubnetId
     publicNetworkAccess: publicNetworkAccess
-    allowedOrigins: [ allowedOrigin ]
+    allowedOrigins: allowedOrigins
```

Comment:
Does this always end up allowing origins even if they don't enable user auth? I dont see a conditional that makes it empty in that case. But maybe my eyes have glazed over.

Reply:
Or maybe we always allowed them anyway?

Reply:
allowedOrigin should be empty if they don't enable user auth. I will double check this, good catch

Reply:
So this is an interesting case
1. We were never enabling the portal origins before due to a bug
2. So unless they explicitly specified an allowed origin, we never enabled cors

My thought is that we want to only add in the portal / login origins if auth is enabled, otherwise just pick the origins they added. I'll make this adjustment